addons/escoria-core/game/core-scripts/esc/commands/set_direction.gd

```diff
@@ -0,0 +1,75 @@
+# `set_direction object direction_id [wait]`
+#
+# Turns a movable `ESCItem` or `ESCPlayer` to face a given target direction id
+# (between 0 and 3 for a 4-directional character, or between 0 and 7 for an
```

Comment:
Which direction does each ID correspond to? E.g. is 0 meant to be North? East? South? West? etc. Or does it correspond to the animations setup for the item/player? We should be specific about this in the docstring/comments.